src/main/kotlin/org/malachite/estella/task/api/TaskExceptionHandler.kt

```diff
@@ -0,0 +1,21 @@
+package org.malachite.estella.task.api
+
+import org.malachite.estella.commons.Message
+import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.task.domain.InvalidTestFileException
+import org.malachite.estella.task.domain.TaskNotFoundException
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+@ControllerAdvice
+class TaskExceptionHandler {
+    @ExceptionHandler(TaskNotFoundException::class)
+    fun handleTaskNotFound(ex: TaskNotFoundException): ResponseEntity<Message> =
+        OwnResponses.NO_RESOURCE("This task doesn't exist or you don't have access to it")
+
+    @ExceptionHandler(InvalidTestFileException::class)
+    fun handleInvalidTestFile(ex: InvalidTestFileException): ResponseEntity<Message> =
+        ResponseEntity(Message("You've provided malformed test file"), HttpStatus.BAD_REQUEST)
```

Comment:
Nie wiem, czy nie fajnie byłoby rzucić, co takiego było złego, jeśli mamy do takich informacji dostęp

Reply:
Da się na pewno ulepszyć ten message, ale:
- widziałem błędy jakie są rzucane przez tę deserializacyjną libkę i są dość kiepskie
- ogólnie jeśli dostałeś coś takiego, to oznacza że został podupcony format więc sobie popraw
Myślę, że the best I can do is napisać jaki ma być ten format